org.mixedrealitytoolkit.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -525,12 +551,16 @@ protected struct LogicImplementation
 
         private bool ShouldSmooth => (IsGrabSelected && SmoothingNear) || (!IsGrabSelected && SmoothingFar);
 
+        private bool UseForces => rigidBody != null && !rigidBody.isKinematic;
+
         private Rigidbody rigidBody;
 
         private bool wasGravity = false;
 
         private bool wasKinematic = false;
 
+        private MovementType currentRigidbodyMovementType;
```

Comment:
I wonder if we should expose this value to subclasses?  Make a property instead?

```cs
/// <summary>
/// The selected <see cref="XRBaseInteractable.MovementType"/>of the rigid body during manipulation.
/// </summary>
/// <remarks>
/// This defaults to <see cref="DefaultRigidbodyMovementType"/>, but can be override 
/// by an interactor with <see cref="XRBaseInteractor.selectedInteractableMovementTypeOverride"/>.
/// </remarks>
protected MovementType CurrentRigidbodyMovementType { get; private set; } = // some default value
```

Reply:
I like that idea, useful to subclasses.  Done.